packages/amplication-client/src/Purchase/PurchasePage.tsx

```diff
@@ -78,6 +79,11 @@ const PurchasePage = (props) => {
     });
   }, [openHubSpotChat, currentWorkspace.id]);
 
+  const handleDowngradeClick = useCallback(() => {
+    history.push("?downgrade=true");
```

Comment:
@EugeneTseitlin, I assume the URL is required for Hubspot to identity the specific chat flow to open? if that is correct, please make sure it is documented here in the code otherwise, someone in the future can easily remove it without understanding the need.

Reply:
also, please add event "PurchaseDowngradeClick"

Reply:
> also, please add event "PurchaseDowngradeClick"

This event has already exist.

Reply:
@EugeneTseitlin 
comments in code

Reply:
comments are added, event that you meant is "PricingPageCTAClick" with "intentionType": "dowgrade" is issed on line 111.